hail/python/hail/experimental/datasets.json

```diff
@@ -27,12 +71,57 @@
             "us": "gs://hail-datasets-us/1000_Genomes_autosomes.phase_3.GRCh38.mt"
           }
         },
+        "version": "phase_3_autosomes"
+      },
+      {
+        "reference_genome": "GRCh38",
+        "url": {
+          "aws": {
+            "us": "s3://hail-datasets-us-east-1/1000_Genomes_chrX.phase_3.GRCh38.mt"
+          },
+          "gcp": {
+            "eu": "gs://hail-datasets-eu/1000_Genomes_chrX.phase_3.GRCh38.mt",
+            "us": "gs://hail-datasets-us/1000_Genomes_chrX.phase_3.GRCh38.mt"
+          }
+        },
+        "version": "phase_3_chrX"
+      },
+      {
+        "reference_genome": "GRCh38",
+        "url": {
+          "aws": {
+            "us": "s3://hail-datasets-us-east-1/1000_Genomes_chrY.phase_3.GRCh38.mt"
+          },
+          "gcp": {
+            "eu": "gs://hail-datasets-eu/1000_Genomes_chrY.phase_3.GRCh38.mt",
+            "us": "gs://hail-datasets-us/1000_Genomes_chrY.phase_3.GRCh38.mt"
+          }
+        },
+        "version": "phase_3_chrY"
```

Comment:
I think there's an open question about how best to represent multi-part datasets. For now, let's stick with the old style for symmetry's sake.

Reply:
Cool, just changed it so it's like `1000_Genomes_autosomes` -> `1000_Genomes_Retracted_autosomes` for each retracted phase 3 GRCh38 dataset